src/modules/esl-carousel/plugin/dots/README.md

```diff
@@ -0,0 +1,116 @@
+# ESLCarouselNavDots
+
+`ESLCarouselNavDots` is auxiliary custom tag for ESLCarousel that provides dots navigation control for the carousel. 
+
+`ESLCarouselNavDots` relay on slide groups(sets), so one dot is created for each slide group. 
+Slide group is a set of slides that are displayed together in the carousel (you can control it by `count` attribute of carousel).
+
+Please note, the plugin does not render if the carousel not found or expected amount of dots is less than 2.
+The `esl-carousel-dots` element does not allow any children except `esl-carousel-dot` elements.
+You can use CSS `:empty` selector to style as inactive state indicator.
```

Comment:
If it is in CSS block flow, it might take up space and shift the elements. 
We might want to hide it in a different way to preserve size and prevent visual jumps. 
That's the first thing that came to my mind.

I probably add an example to the doc as well.